config/sti.properties

```diff
@@ -2,15 +2,16 @@
 #	FILE PATHS 	   #
```

Comment:
Tento soubor - tj. vase nastaveni necommitujte do githubu - zmente jen (v te vasi branch) odpovidajicim zpusobem config.properties.sample

Reply:
done - I reverted the `sti.properties` file and added a new `sti.properties.sample` file, which contains the required change (path to the `ml.properties` file)